zero_bin/rpc/src/lib.rs

```diff
@@ -102,28 +84,34 @@ where
         })
         .take(PREVIOUS_HASHES_COUNT + 1)
         .filter(|i| *i >= 0)
+        .chunks(2)
+        .into_iter()
+        .map(|mut chunk| {
+            // We convert to tuple of (current, Option previous block)
+            if let Some(first) = chunk.next() {
+                let second = chunk.next();
+                (first, second)
+            } else {
+                panic!("not possible according to itertools::Iterator::chunks definition")
+            }
```

Comment:
Random nit that you can ignore, but wdyt about using expect here?

```rust
let first = chunk.next().expect("not possible according to itertools::Iterator::chunks definition");
let second = chunk.next();
(first, second)
```

Reply:
Indeed :)